collections/_article/godotcon-2025-xr-android-recap.md

```diff
@@ -0,0 +1,135 @@
+
+<img alt="Godot XR editor average monthly users" src="/storage/blog/godotcon-2025-android-xr/godot-xr-editor-avg-monthly-users-2025.webp" />
+
+As we stated last year this is only the beginning. We have a lot planned to improve the user experience in order to make the XR editor a powerful, flexible tool for XR and game development, but first we need to improve the reach of the XR editor and make it cross-platform.  
+To that end, we are working to bring the Godot XR editor to **Pico devices\!**
+
+<img alt="Pico device" src="/storage/blog/godotcon-2025-android-xr/pico_device_image.webp" />
+
+Early alpha builds can be found on our [Github release page](https://github.com/godotengine/godot-builds/releases) with support for both **Pico 4 & Pico 4 Ultra** devices.
+
+And we’re not stopping there. We’ve heard loud and clear the feedback and requests for Quest 2 support, and thanks to recent performance improvements courtesy of W4 Games, we’re adding **official support for Quest 2** devices as well\!  
+You can now download the XR editor to your Quest 2 devices straight from the Meta Horizon store\!
+
+# Hybrid Apps
+
+Introduced in late 2023, Hybrid Apps are apps that can run both in **Panel** mode or in **Immersive** mode, and are able to **transition between both modes at runtime**. In Panel mode, the app can run side-by-side with other apps, allowing users to multitask. While in Immersive mode, the app’s experience is the sole focus and the user is fully immersed in it.
+
+The feature provides developers with several benefits:
+
+* Developers can easily add immersive support to existing apps & games  
+* Developers can leverage existing codebase and logic  
+* Users can select the interaction mode they’re most comfortable with
+
+The Godot XR editor is itself a Hybrid App and has been able to leverage this feature to provide a unique development experience, allowing developers to transition back and forth between the editor window and their immersive projects.   
+With the release of [version 4 of the Godot OpenXR Vendors plugin](https://github.com/GodotVR/godot_openxr_vendors/releases/tag/4.0.0-stable), we’re bringing the same capabilities to all developers in Godot 4.4\!
+
+ <video autoplay loop muted playsinline title="Kenney 3D platformer Hybrid App sample">
+  <source src="/storage/blog/godotcon-2025-android-xr/godot-hybrid-app-kenney-3d-platformer.mp4?1" type="video/mp4">
+</video>
+
+By leveraging **v4** of the Godot OpenXR Vendors plugin, developers can build Hybrid Apps straight from the Godot editor. The feature supports all HorizonOS (Quest) devices, with support for Pico devices coming soon\!
+
+The feature dynamically enables / disables itself based on the target platform. For example, **the same Godot project can run as a regular (panel) app when exported to regular Android devices. And it can run as a full Hybrid App when exported to standalone XR devices**.  
```

Comment:
```suggestion
The feature dynamically enables or disables itself based on the target platform. For example, **the same Godot project can run as a regular (panel) app when exported to regular Android devices. And it can run as a full Hybrid App when exported to standalone XR devices**.  
```